site2/docs/security-basic-auth.md

```diff
@@ -73,7 +79,9 @@ To configure brokers to authenticate clients, complete the following steps.
    authenticateOriginalAuthData=true
    ```
 
-2. Set an environment variable named `PULSAR_EXTRA_OPTS` and the value is `-Dpulsar.auth.basic.conf=/path/to/.htpasswd`. Pulsar reads this environment variable to implement HTTP basic authentication.
+   :::note
```

Comment:
If you want to turn step2 into a note, you need to remove the numbering of step1 (numbering is not required since there is only one step left) and the indention of the note.